src/components/GoodsData.ts

```diff
@@ -0,0 +1,51 @@
+import goodsData from '../modules/goods';
+
+export default class GoodsData {
+  static getAllPrice(): number[] {
+    const allPrices: number[] = [];
+    for (let i = 0; i < goodsData.products.length; i++) {
+      allPrices.push(goodsData.products[i].price);
+    }
+    return allPrices;
+  }
+
+  static getMinPrice(): number {
+    const minPrice: number = Math.min(...this.getAllPrice());
+    return minPrice;
+  }
+
+  static getMaxPrice(): number {
+    const maxPrice: number = Math.max(...this.getAllPrice());
+    return maxPrice;
+  }
+
+  static getAllStock(): number[] {
+    const allStock: number[] = [];
+    for (let i = 0; i < goodsData.products.length; i++) {
+      allStock.push(goodsData.products[i].stock);
+    }
+    return allStock;
+  }
+
+  static getMinStock(): number {
+    const minStock: number = Math.min(...this.getAllStock());
+    return minStock;
+  }
+
```

Comment:
you can return without creating a variable
`
static getMinStock(): number {
    return Math.min(...this.getAllStock());
  }
`